src/platform/utilities/data/buildQueryString.js

```diff
@@ -0,0 +1,13 @@
+import appendQuery from 'append-query';
+
+/**
+ * Uses the append-query lib (https://github.com/lakenen/node-append-query) to
+ * assemble and append a valid query string to URLs.
+ *
+ * basePath {String} - Will often be '/' for React app base
+ * searchQuery {Object} {String} - Search parameters in key/value format
+ */
+export default function buildQueryString(basePath, searchQuery) {
```

Comment:
@rianfowler The thinking behind creating a helper was pulling in `appendQuery` once and having it available for `FacilityLocator` and `VAMap`. I've got at least one other instance where this will be beneficial in GI Bill® Comparison Tool, so I can refactor as you've suggested here, or keep as is.